permissions/plugin-aws-sqs.yml

```diff
@@ -0,0 +1,6 @@
+---
+name: "aws-sqs-plugin"
+paths:
+- "org/jenkins-ci/plugins/aws-sqs-plugin"
```

Comment:
Thanks for that, changes made.

- **pom.xml** - changed artifactId to [`aws-sqs-plugin`](https://github.com/jenkinsci/aws-sqs-plugin/blob/master/pom.xml#L15)
- **PR/plugin-aws-sqs.yml** - changed path to [match the instructions](https://github.com/jenkins-infra/repository-permissions-updater/pull/49/files#diff-f13797a5b8fe26791b26695e34639433R4)